service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/SpecimenMapperTest.java

```diff
@@ -1,4 +1,4 @@
-package uk.nhs.adaptors.gp2gp.ehr.mapper;
+package uk.nhs.adaptors.gp2gp.ehr.mapper.diagnosticreport;
```

Comment:
Moving `SpecimenMapperTest` to the "diagnosticreport" sub-folder in the unit tests directory.